packages/editor/src/plugins/text/hooks/use-slate-render-handlers.tsx

```diff
@@ -108,7 +110,7 @@ export const useSlateRenderHandlers = ({
           mayManipulateSiblings
             ? () => {
                 ReactEditor.focus(editor)
-                editor.insertText('/')
+                setShowSuggestions?.(true)
```

Comment:
The dependency array is now missing the expected `setShowSuggestions` dependency.

Reply:
Fixed